moto/support/responses.py

```diff
@@ -24,3 +24,57 @@ def refresh_trusted_advisor_check(self):
         status = self.support_backend.refresh_trusted_advisor_check(check_id=check_id,)
 
         return json.dumps(status)
+
+    def resolve_case(self):
+        case_id = self._get_param("caseId")
+        resolve_case_response = self.support_backend.resolve_case(case_id=case_id,)
+        return json.dumps(resolve_case_response)
+
+    def create_case(self):
+        subject = self._get_param("subject")
+        service_code = self._get_param("serviceCode")
+        severity_code = self._get_param("severityCode")
+        category_code = self._get_param("categoryCode")
+        communication_body = self._get_param("communicationBody")
+        cc_email_addresses = self._get_list_prefix("ccEmailAddresses.member")
+        language = self._get_param("language")
+        issue_type = self._get_param("issueType")
+        attachment_set_id = self._get_param("attachmentSetId")
+        create_case_response = self.support_backend.create_case(
+            subject=subject,
+            service_code=service_code,
+            severity_code=severity_code,
+            category_code=category_code,
+            communication_body=communication_body,
+            cc_email_addresses=cc_email_addresses,
+            language=language,
+            issue_type=issue_type,
+            attachment_set_id=attachment_set_id,
+        )
+
+        return json.dumps(create_case_response)
+
+    def describe_cases(self):
+        case_id_list = self._get_param("caseIdList")
+        display_id = self._get_param("displayId")
+        after_time = self._get_param("afterTime")
+        before_time = self._get_param("beforeTime")
+        include_resolved_cases = self._get_param("includeResolvedCases")
```

Comment:
Reading the docs, this param defaults to False. 
Looks like `include_communications` can default to True